code/src/test/java/com/google/sps/storage/BusinessTest.java

```diff
@@ -0,0 +1,23 @@
+package com.google.sps.storage;
+
+import org.junit.Test;
+import org.junit.Assert;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+ 
+@RunWith(JUnit4.class)
+public final class BusinessTest {
+
+  @Test
+  public void createBusiness() {
+    Business business = new Business("Test");
+    Assert.assertFalse(business.contactedInLastWeek());
+  }
+
+  @Test
+  public void contactBusiness() {
+    Business business = new Business("Test");
```

Comment:
using the arrange, act, assert model, each of these lines should have a newline between them.

Reply:
done